modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -13,9 +34,87 @@ export default function ProfileViewBasics({ profile }) {
+          {profile.languages.map(code => <li key={code}>{t(languages[code], { ns: 'languages' }) || code}</li>)}
+        </ul>
+      </div>}
+      { (hasConnectedAdditionalSocialAccounts() || profile.extSitesBW || profile.extSitesCS || profile.extSitesWS) &&
+        <div className="profile-sidebar-section">
+          <h4 id="profile-networks" aria-label="Member in other networks">
+            {t('Elsewhere')}
+          </h4>
+          <ul className="social-profiles list-unstyled"
+            aria-describedby="profile-networks">
+            {/*
+              Facebook profile link is hidden here until issue with their API gets resolved
+              See https://github.com/Trustroots/trustroots/issues/237
+            */}
+            {profile.additionalProvidersData && Object.keys(profile.additionalProvidersData).map(
+              (key, indeks) =>
+              {return (key !== 'facebook') && <li className="social-profile" key={indeks}><i className={`social-profile-icon icon-fw icon-lg icon-${key}`}></i>
+                <a rel="noopener"
+                  className="social-profile-handle text-capitalize"
+                  href={socialAccountLink(key, profile.additionalProvidersData[key])}>{key}
+                </a>
+              </li>;
+              }
+            )}
+
+            {profile.extSitesBW &&
+            <li className="social-profile">
+              <i className="social-profile-icon icon-fw icon-lg icon-bw"></i>
+              <a rel="noopener"
+                className="social-profile-handle"
+                href={`https://www.bewelcome.org/members/${profile.extSitesBW}`}>
+                BeWelcome
+              </a>
+            </li>}
+
+            {profile.extSitesCS &&
+            <li className="social-profile">
+              <i className="social-profile-icon icon-fw icon-lg icon-cs"></i>
+              {/*
+                Link here has `noreferrer` because;
+                https://github.com/Trustroots/trustroots/issues/464
+              */}
+              <a rel="noreferrer noopener"
+                className="social-profile-handle"
+                href={`https://www.couchsurfing.com/people/${profile.extSitesCS }`}>
+                Couchsurfing
+              </a>
+            </li>}
+
+            {profile.extSitesWS &&
+            <li className="social-profile">
+              <i className="social-profile-icon icon-fw icon-lg icon-warmshowers"></i>
+              <a className="social-profile-handle"
+                href={`https://www.warmshowers.org/${isWarmshowersId ? 'user' : 'users' }/${profile.extSitesWS}`}>
+                Warmshowers
+              </a>
+            </li>}
+          </ul>
+        </div>}
   </>) }</NamespacesConsumer>);
 };
 
 ProfileViewBasics.propTypes = {
-  profile: PropTypes.object
+  profile: PropTypes.object,
+  hasConnectedAdditionalSocialAccounts: PropTypes.func,
+  isWarmshowersId: PropTypes.func,
+  socialAccountLink: PropTypes.func
```

Comment:
I wonder if we could put these utility functions elsewhere and import them to both here in React component and also in Angular controller, so that we wouldn't need to pass them down to React component from the Angular controller.

@mrkvon suppose we cannot `import` in Angular controllers just yet?

These specific ones are so small that I wouldn't really mind even just to duplicate them for the time being.

I'd put them in `modules/users/client/utils/networks.js`